deploy/overlays/karnataka/kustomization.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -private
+namespace: website
+
+resources:
+- ../../base
+
+configMapGenerator:
+- behavior: create
+  literals:
+  - flaskEnv=autopush
+  - secretProject=datcom-karnataka
+  name: website-configmap
+- behavior: create
+  literals:
+  - mixerProject=datcom-karnataka
+  - serviceName=website-esp.endpoints.datcom-karnataka.cloud.goog
+  name: mixer-configmap
+
+patchesStrategicMerge:
+- |-
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: website-app
+  spec:
+    replicas: 9
+images:
```

Comment:
This is an updated version of the file. Can you copy it again from other folder, edit it and commit?

I already have a fix that do not update this file but this PR has not sync pass that..

Reply:
done